.github/workflows/code-quality.yaml

```
@@ -78,7 +78,7 @@ jobs:
             -   name: Setup PHP
                 uses: shivammathur/setup-php@v2
                 with:
-                    php-version: '8.1'
+                    php-version: ${{ matrix.php-version }}
```

```suggestion
                    php-version: 8.4
```

and remove `php-version` from `matrix`